src/coreclr/src/tools/aot/crossgen2/Program.cs

```diff
@@ -347,6 +348,27 @@ private int Run()
                         }
                     }
 
+                    foreach (string inputFilePath in allInputFilePaths.Values)
```

Comment:
Would you mind breaking this out into a method? Ideally `Run` will get smaller over time

Reply:
Fixed in 3rd commit.